crates/fuel-streams-core/src/outputs/subjects.rs

```diff
@@ -0,0 +1,311 @@
+use fuel_core_types::fuel_types::{Address, Bytes32, ContractId};
```

Comment:
This is making your tests failing because you are using the wrong types here; you need to use these types, but import from the crate instead
```suggestion
use crate::types::{Address, Bytes32, ContractId};
```